e2e/1_16-exceptions.yml

```diff
@@ -0,0 +1,20 @@
+# This metrics are not present from 1.16
+except_metrics:
+# Added from 1.23
+- k8s.apiserver.currentInflightRequestsReadOnly
```

Comment:
Previously, the `e2e.yaml` workflow used the `1_22` exceptions file for testing K8s `1.16`. All of these metric exceptions are from the `1_22` file.